### ISSUE TITLE
Suite B (RFC6460), libressl uses SHA1

### ISSUE THREAD

**Issue by Safari77** [libressl-portable/portable#143]
with TLS_ECDHE_ECDSA suites, I get these with libressl + nginx:

```
Peer signing digest: SHA1
Server Temp Key: ECDH, secp256k1, 256 bits
```

```
Peer signing digest: SHA1
Server Temp Key: ECDH, P-384, 384 bits
```

RFC6460 specifies usage of SHA-256 and SHA-384.
The check seems to be `ssl/t1_lib.c:tls1_check_cert_param()` in openssl 1.0.2.

**Comment by 4a6f656c** [comment]
I suspect this will have been related to the weak digest fallback that occurred with SNI, which was fixed in c2f8e0c15b8c5da6f28e5c2eb806acfdaf300557.

**Comment by 4a6f656c** [comment]
@Safari77 - I am going to close this on the basis that it should already be fixed - if this is not the case please reopen.
